pkgs/tools/admin/auth0-cli/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "auth0-cli";
+  version = "0.11.10";
+
+  src = fetchFromGitHub {
+    owner = "auth0";
+    repo = "auth0-cli";
+    rev = "v${version}";
+    hash = "sha256-1/T2hpSNamorWFuaSBoLsGpe9I06HGew9S3yJsDLmLQ=";
+  };
+
+  vendorHash = "sha256-d9ZwK/LAZGgeagGsg3bGYnVykfQcCLUex0pe/PUCtkA=";
+
```

Comment:
Need to set version via ldflags

something like:

`ldflags = [ "-s" "-w" "-X github.com/auth0/auth0-cli/internal/buildinfo.Version=v${version}" ];`

Check if upstream uses a v prefix or not

Reply:
Done